grpc.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |s|
   s.require_paths = %w( src/ruby/lib src/ruby/bin src/ruby/pb )
   s.platform      = Gem::Platform::RUBY
 
-  s.add_dependency 'google-protobuf', '~> 3.1'
+  s.add_dependency 'google-protobuf', '~> 3.7.2.rc.2'
```

Comment:
@apolcyn  what's the right way to specify the 3.7.2 prerelease package? 
https://rubygems.org/gems/google-protobuf/versions/3.7.0.rc.2-x86_64-linux

Reply:
I'm getting this error: https://source.cloud.google.com/results/invocations/aeae23e1-b6c7-412f-8e58-02ac05cced22/targets/github%2Fgrpc%2Faggregate_tests/tests;group=aggregate_tests;test=run_tests_ruby_linux_dbg_native;row=3

Reply:
Sorry. Can we depend on `3.7.0.rc.2`? I think that's related to the below error. `0.rc.2` is the version that's published on rubygems.org

Reply:
Ah, it was a typo. I actually rolled back that change from the PR because I wasn't sure how to fix. Oddly, it still fails.